components/datetime/src/neo_marker.rs

```diff
@@ -1648,7 +1735,11 @@ impl DateTimeNamesMarker for NeoDateTimeComponents {
     type WeekdayNames = datetime_marker_helper!(@names/weekday, yes);
     type DayPeriodNames = datetime_marker_helper!(@names/dayperiod, yes);
     type ZoneEssentials = datetime_marker_helper!(@names/zone/essentials, no);
-    type ZoneGenericShortNames = datetime_marker_helper!(@names/zone/genericshort, no);
+    type ZoneExemplarCities = datetime_marker_helper!(@names/zone/exemplar, no);
+    type ZoneGenericLong = datetime_marker_helper!(@names/zone/generic_long, no);
+    type ZoneGenericShort = datetime_marker_helper!(@names/zone/generic_short, no);
+    type ZoneSpecificLong = datetime_marker_helper!(@names/zone/specific_long, no);
+    type ZoneSpecificShort = datetime_marker_helper!(@names/zone/specific_short, no);
```

Comment:
Not 100% sure yet whether these should be yes or no; I think they probably should be yes, but I'll find out when writing more end-to-end time zone tests